src/peergos/shared/user/fs/FileWrapper.java

```diff
@@ -418,11 +418,6 @@ public boolean isLink() {
         return linkPointer.isPresent();
     }
 
-    @JsMethod
```

Comment:
What is the alternative?

Reply:
UserContext.sharedWith(path), but I can add something that return a boolean

Reply:
Actually I removed the boolean method, see replies to other comment